From 6519ad6e6f92a4310fbaab50f074d5bbad27b329 Mon Sep 17 00:00:00 2001
From: K. Y. Srinivasan <kys@microsoft.com>
Date: Thu, 22 Sep 2011 12:40:50 -0700
Patch-mainline: staging-next-20111001 ?
Subject: [PATCH] Staging: hv: Update the TODO file

Now that all vmbus audit related comments have been addressed,
update the TODO file to reflect this.

Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Haiyang Zhang <haiyangz@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/TODO |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/staging/hv/TODO
+++ b/drivers/staging/hv/TODO
@@ -1,6 +1,4 @@
 TODO:
-	- audit the vmbus to verify it is working properly with the
-	  driver model
 	- audit the network driver
 	  - checking for carrier inside open is wrong, network device API
             confusion??
